ci/terraform/oidc/api-gateway.tf

```diff
@@ -1020,4 +1022,39 @@ resource "aws_api_gateway_integration" "orch_authorisation_integration" {
   type                    = "AWS_PROXY"
   integration_http_method = "POST"
   uri                     = "arn:aws:apigateway:eu-west-2:lambda:path/2015-03-31/functions/arn:aws:lambda:eu-west-2:${var.orch_account_id}:function:${var.orch_authorisation_name}:latest/invocations"
-}
\ No newline at end of file
+}
+
+resource "aws_api_gateway_resource" "orch_processing_identity_resource" {
+  count       = var.orch_processing_identity_enabled ? 1 : 0
+  rest_api_id = aws_api_gateway_rest_api.di_authentication_api.id
+  parent_id   = aws_api_gateway_rest_api.di_authentication_api.root_resource_id
+  path_part   = "processing-identity"
+  depends_on = [
+    module.processing-identity
+  ]
+}
+
+resource "aws_api_gateway_method" "orch_processing_identity_method" {
+  count            = var.orch_processing_identity_enabled ? 1 : 0
+  rest_api_id      = aws_api_gateway_rest_api.di_authentication_api.id
+  resource_id      = aws_api_gateway_resource.orch_processing_identity_resource[0].id
+  http_method      = "POST"
+  api_key_required = true
+  depends_on = [
+    aws_api_gateway_resource.orch_processing_identity_resource
+  ]
+  authorization = "NONE"
+}
+
+resource "aws_api_gateway_integration" "orch_processing_identity_integration" {
+  count       = var.orch_jwks_enabled ? 1 : 0
+  rest_api_id = aws_api_gateway_rest_api.di_authentication_api.id
+  resource_id = aws_api_gateway_resource.orch_processing_identity_resource[0].id
+  http_method = aws_api_gateway_method.orch_processing_identity_method[0].http_method
+  depends_on = [
+    aws_api_gateway_resource.orch_processing_identity_resource
+  ]
+  type                    = "AWS_PROXY"
+  integration_http_method = "POST"
+  uri                     = "arn:aws:apigateway:eu-west-2:lambda:path/2015-03-31/functions/arn:aws:lambda:eu-west-2:${var.orch_account_id}:function:${var.orch_processing_identity_name}:latest/invocations"
+}
```

Comment:
Is this correct here ? This lambda is exposed on the frontend api ? If so will need to be moved and rest_api_id corrected to reference frontend.